frontend/src/app/modules/shared/modules/notification/modal/accept/accept-notification-modal.component.ts

```diff
@@ -45,8 +46,16 @@ export class AcceptNotificationModalComponent {
     this.formGroup = new UntypedFormGroup({ reason: this.textAreaControl });
   }
 
-  public show(notification: Notification): void {
+  public show(notification: Notification, translationContext: TranslationContext): void {
     this.notification = notification;
+
+    if(translationContext === TranslationContext.COMMONINVESTIGATION) {
```

Comment:
I think we could refactor it a bit, and make lighter, if we introduce this changes:

- TranslationContext will contain as values context names that we need to prepare key for get translations:
```suggestion
   export enum TranslationContext {
  COMMONINVESTIGATION = "commonInvestigation",
  COMMONALERT = "commonAlert"
}
```
- then we can simply a logic below, just putting:
```suggestion
    this.translationContext = translationContext;
```